sdk/python/kfp/v2/components/experimental/component_spec_test.py

```diff
@@ -14,109 +14,115 @@
 """Tests for kfp.v2.components.experimental.component_spec."""
 
 import unittest
+from unittest.mock import patch, mock_open
 
 from kfp.v2.components.experimental import component_spec
 
 
 class ComponentSpecTest(unittest.TestCase):
 
-  def test_component_spec_with_duplicate_inputs_outputs(self):
-    with self.assertRaisesRegex(ValueError,
-                                'Non-unique input name "dupe_param"'):
-      component_spec.ComponentSpec(
-          name='component_1',
-          implementation=component_spec.ContainerSpec(
-              image='alpine',
-              commands=[
-                  'sh',
-                  '-c',
-                  'set -ex\necho "$0" > "$1"',
-                  component_spec.InputValuePlaceholder('dupe_param'),
-                  component_spec.OutputPathPlaceholder('output1'),
-              ],
-          ),
-          input_specs=[
-              component_spec.InputSpec(name='dupe_param', type=str),
-              component_spec.InputSpec(name='dupe_param', type=int),
-          ],
-          output_specs=[
-              component_spec.OutputSpec(name='output1', type=str),
-          ],
-      )
+  # Uncomment when init method for ComponentSpec is done.
```

Comment:
nit:  a cleaner way could be using [`@unittest.skip`](https://docs.python.org/3/library/unittest.html#unittest.skip)